vagrant/README.md

```diff
@@ -0,0 +1,138 @@
+Vagrant provider for jclouds
+============================
+
+Building
+--------
+
+1. Vagrant bindings
+  * `git clone https://github.com/neykov/vagrant-java-bindings`
+  * `cd vagrant-java-bindings`
+  * `mvn clean install`
+  * Copy `target/vagrant-java-bindings-0.0.1-SNAPSHOT.jar` to jcloud's classpath
+2. Vagrant provider
+  * `git clone https://github.com/neykov/jclouds-vagrant`
+  * `cd jclouds-vagrant/vagrant`
+  * `mvn clean install`
+  * Copy `target/vagrant-2.0.0-SNAPSHOT.jar` to jcloud's classpath
+
+Using in Apache Brooklyn
+------------------------
+
+Download a Vagrant box:
+```
+vagrant box add ubuntu/trustry64
```

Comment:
possibly in my case it was a combination of the above and a previously downloaded `ubuntu/trusty64` on my machine, thx